frontend/packages/prompt-test/src/index.ts

```diff
@@ -19,7 +19,7 @@ interface FileChange {
 
 interface PromptVars {
   docsContent: string
-  schemaFiles: SchemaFile[]
+  schemaFile: SchemaFile[]
```

Comment:
This file isn't critical, so no need to spend too much time on it. I’m planning to revisit and revise it thoroughly later anyway!

Reply:
@hoshinotsuyoshi Thanks, I tried to fix it, but let me ignore it for now, assuming I'm moving to langfuse 🙏🏻